website/package.json

```diff
@@ -30,23 +29,24 @@
     "gatsby-source-filesystem": "^2.11.0",
     "gatsby-transformer-json": "^2.11.0",
     "gatsby-transformer-remark": "^2.16.0",
-    "gatsby-transformer-sharp": "^2.12.0",
+    "jquery": "^3.6.0",
```

Comment:
Why we need to include jquery and popper.js now?

Reply:
@attfarhan they were coming up as a peer dependency that was missing.